modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1678,7 +1678,7 @@
                 "format": "java",
                 "type": "Object",
                 "annotations": {
-                  "typeId": "org.mule.runtime.core.routing.AggregationStrategy",
+                  "typeId": "org.mule.runtime.core.internal.routing.AggregationStrategy",
```

Comment:
@gsfernandes in this case, not, is not necessary for tooling to have access to the class, only rt should require it. That said, all the parameters should be of types exposed as part of the api, none internal.

sidenote: with that definition of "custom-aggregation-strategy" tooling is not going to be able to construct the element correctly